docs/installguide/release_notes.rst

```diff
@@ -10,7 +10,7 @@ General
 Python 2.6 is no longer supported. It *may* still work, but we are no longer actively supporting it.
 Other known issues:
 
-* The latest OSX version (EL Capitan) is not yet supported.
+* The latest OSX version (EL Capitan) is not yet supported. KA Lite is officially supported on OS X 10.8 - 10.10.
```

Comment:
But these are 0.15.0 release notes. We should add an "El capitan is now supported" for 0.16.0.